pkg/network/usm/ebpf_main.go

```diff
@@ -544,6 +544,18 @@ func (e *ebpfProgram) getProtocolStats() map[protocols.ProtocolType]interface{}
 	return ret
 }
 
+// releaseProtocolStats release stats objects for enabled protocols.
+func (e *ebpfProgram) releaseProtocolStats(conns *network.Connections) {
+	for _, protocol := range e.enabledProtocols {
+		switch protocol.Instance.Type() {
+		case protocols.HTTP:
+			for _, rs := range conns.HTTP {
+				rs.ReleaseStats()
+			}
+		}
+	}
+}
```

Comment:
Having USM main entrypoint getting NPM representation of connections, just to free internal state is not the right way to do it.

Reply:
I intend to change this in the next phase, this approach allowed me to keep the changes at this stage minimal.